tests/formatter.test.ts

```diff
@@ -615,4 +615,11 @@ public type T = {
         await expectFormatted('?#abc\n');
         expect(await format('? #abc')).toEqual('?#abc\n');
     });
+
+    test('parenthesized `with` expression', async () => {
+        await expectFormatted('(x with a = 1);\n');
```

Comment:
```suggestion
        await expectFormatted('(x with a = 1)\n');
```
These are never followed by a semi, as they are not expressions/statements. Instead they prefix expressions.

Reply:
Also maybe test `        await expectFormatted('(with a = 1)\n');`

Reply:
Oh I see what you're saying; I'll update the tests so that it includes the enclosing actor, since these cases are assumed to be (invalid) standalone expressions which end in trailing semicolons.

Reply:
Yeah, see:
``` Motoko
$ rlwrap moc
Motoko compiler (source 0.13.7-202-g68eddf78f-dirty)
> (x with a = 1; b = 2);
stdin:1.22-1.23: syntax error [M0001], unexpected token ';', expected one of token or <phrase> sequence:
  <obj_or_class_dec>
  async <exp_nest>
  <exp_post(ob)> <inst> <exp_nullary(ob)>
```